commons/net/http/withLogging_middleware.go

```
@@ -0,0 +1,253 @@
+		return context.Background()
+	}
+
+	return ctx
+}
+
+func getContextHeaderID(ctx context.Context) string {
+	if ctx == nil {
+		return ""
+	}
+
+	_, _, headerID, _ := observability.NewTrackingFromContext(ctx)
```

golangci-lint (2.12.2)

[error] declaration has 3 blank identifiers

(dogsled)